lib/Backends/Habana/HabanaDeviceManager.cpp

```diff
@@ -26,12 +27,20 @@
 using namespace glow;
 using namespace glow::runtime;
 
+namespace glow {
+namespace runtime {
+
+uint64_t GlowHabanaMemory = std::numeric_limits<uint64_t>::max();
```

Comment:
would it make sense to have more conservative value (7516192768; // 7GB)?

Reply:
Probably for now.  I was thinking that once the actual get-mem-info API lands, we could use min(command line param, actual reported device memory)